src/hooks/useMaxInputBalance.ts

```diff
@@ -2,9 +2,10 @@ import { useCallback, useState } from 'react';
 import useGas from './useGas';
 import { ethereumUtils } from '@/utils';
 import { ParsedAddressAsset } from '@/entities';
+import { convertStringToNumber } from '@/helpers/utilities';
 
 export default function useMaxInputBalance() {
-  const [maxInputBalance, setMaxInputBalance] = useState(0);
+  const [maxInputBalance, setMaxInputBalance] = useState<number>(0);
```

Comment:
can we just switch the default for the state to '0' and then you won't need to do any of the other changes as the ethereumutils for handling numbers should be returning string also